updater/reports/ReportPRUsage.py

```diff
@@ -35,8 +35,8 @@ def query(self, date):
 		query = '''
 			SELECT
 				"''' + str(date) + '''" AS date,
-				ROUND(100*COUNT(pr.repository_id)/COUNT(active.repository_id)) AS "Pull request usage in %",
-				ROUND(100*COUNT(status.repository_id)/COUNT(active.repository_id)) AS "Status usage in %"
+				ROUND(100*COUNT(pr.repository_id)/COUNT(active.repository_id)) AS "pull request usage [%]",
+				ROUND(100*COUNT(status.repository_id)/COUNT(active.repository_id)) AS "status usage [%]"
```

Comment:
are these names used anywhere? just to make sure we don't break anything here ;-)

Reply:
Yes, these are the labels shown in this chart:

![screenshot from 2017-10-10 16-27-16](https://user-images.githubusercontent.com/3244280/31391825-e80222a2-add7-11e7-89f3-9389518172f2.png)

Reply:
It’s the actual results if I see that correctly, and not just intermediate subquery results.